frameworks/kafka/docs/upgrade.md

```diff
@@ -1,25 +1,25 @@
 ---
-post_title: Upgrade Kafka from 0.10.2.1 to 0.11.0.0
+post_title: Upgrade Kafka from 0.10.2.x to 0.11.0.0
 menu_order: 10
 feature_maturity: preview
 enterprise: 'no'
 ---
 
-# Upgrade Kafka from 0.10.2.1 to 0.11.0.0
+This document explains how to upgrade the DC/OS Kafka service from version 0.10.2.x to 0.11.0.0.
 
-
-This document explains how to upgrade the DC/OS Kafka service from 0.10.2.x to 0.11.0.0 version. Refer to the Kafka [upgrade](https://kafka.apache.org/documentation/#upgrade) documentation for more information.
+- Refer to the Kafka [upgrade](https://kafka.apache.org/documentation/#upgrade) documentation for more information.
 
 - Information about the new message format in 0.11.0.0 is available [here](https://kafka.apache.org/documentation/#upgrade_11_message_format).
 
-- Kafka 0.11.0.0 introduces idempotent and transactional capabilities. These new 0.11.0.0 features require the new messaging format, and so will not work on older formats. 
+- 0.11.0.0 clients can communicate with older brokers, such as 0.10.2.x brokers. But Kafka 0.11.0.0 introduces idempotent and transactional capabilities. 0.11.0.0 clients are required for these new features. 
+
+# Overview
 
-- 0.11.0.0 clients can communicate with older brokers, such as 0.10.2.x brokers. However 0.11.0.0 clients are required for these new features. 
+This is an overview of the rolling upgrade process, which guarantees no downtime during the upgrade. The process for upgrading Kafka with DC/OS is documented in detail below.
```

Comment:
s/which guarantees no downtime during the upgrade//

While this is our goal, we should not say we guarantee it.